ublox_derive/src/output.rs

```diff
@@ -133,6 +170,24 @@ pub fn generate_recv_code_for_packet(pack_descr: &PackDesc) -> TokenStream {
                     #(#get_value_lines)*
                 }
             });
+            if f.map.flatten {
```

Comment:
more macros ! ?

ive managed to avoid the manual implementation of the esfmeas so we can keep the flatten macro...
but this is really spaghetti

Reply:
idk if theres a way to fetch the fields given the `Type` , was trying that approach but decided to go for just this using strings.

was also trying to create derive macro for the struct EsfMeasInfo to fetch all the fields but apparently rust doesnt like a static mut hashmap globally - (also maybe bad for keeping that just to create all these derived functions?)